website/redirects.js

```diff
@@ -0,0 +1,1200 @@
+module.exports = [
+  {
+    source: '/api-docs/index(.html)?',
```

Comment:
I don't think we should, but it's a simple change if we wanted to do that:
1. A catch-all makes this file less explicit because we remove the redirects that explicitly need support for a `.html` variant
2. A catch-all would enable a very large number of `.html` routes in ways that were never intended, and that could include links from external sources. So essentially we may be opening ourselves up to supporting a much larger class of routes than we want.